pkgs/by-name/hx/hxn/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hxn";
```

If the package directory is changed I suggest changing the package name too:
```suggestion
  pname = "haylxon";
```

---

Thanks for all the suggestions, all your suggestions makes sense. Yes, `haxylon` is the name, I thought nixpkgs required the bin name.

---

Changing the name to `haxylong` breaks the build, hence the pname should be `hxn` and the by-name location must me `hx/hxn`.